app/Http/Controllers/GroupController.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Group;
+use Validator;
+use Illuminate\Http\Request;
+use App\Http\Resources\Group as GroupResource;
+
+
+class GroupController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $groups = Group::all();
+
+        return GroupResource::collection($groups);
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        Validator::make($request->all(), [
+            'name' => 'required|unique:groups,name',
+            'description' => 'nullable',
+        ])->validate();
+
+        $timeNow = now();
+        $group = Group::create([
+            'name' => $request->name,
+            'description' => $request->description,
+            'updated_at' => $timeNow,
```

Comment:
You can remove the updated_at and created_at from here.
These fields will be filled automatically.

Reply:
If you did this also in AdminController, please also change there